k2/csrc/fsa_utils.h

```diff
@@ -271,18 +272,19 @@ Array1<FloatType> GetBackwardScores(
                          from
        @param [in] forward_scores  The state-level forward scores, which
                         should have been computed by GetForwardScores() with
-                        the same `fsas` and `log_semiring`
+                        the same `fsas` and `log_semiring` that
```

Comment:
Can `log_semiring` be removed from the comment?

Reply:
Will remove the inverted comma around `log_semiring`.  But let's keep log_semiring for clarity.